Firmware/language_pt.h

```diff
@@ -0,0 +1,343 @@
+#define MSG_CONTROL                         "Controlo"
+#define MSG_MIN                             " \002 Min"
+#define MSG_MAX                             " \002 Max"
+#define MSG_FACTOR                          " \002 Fator"
+#define MSG_TEMPERATURE                     "Temperatura"
+#define MSG_MOTION                          "Movimento"
+#define MSG_VOLUMETRIC                      "Filamento"
+#define MSG_VOLUMETRIC_ENABLED		        "E em mm3"
+#define MSG_STORE_EPROM                     "Guardar na mem."
+#define MSG_LOAD_EPROM                      "Ler da memoria"
+#define MSG_RESTORE_FAILSAFE                "Rest. Val. Segur."
+#define MSG_REFRESH                         "\xF8" "Atualizar"
+#define MSG_WATCH                           "Ecrã inform."
+#define MSG_TUNE                            "Afinar"
+#define MSG_PAUSE_PRINT                     "Pausa impressao"
+#define MSG_RESUME_PRINT                    "Retomar impressao"
+#define MSG_STOP_PRINT                      "Parar impressao"
+#define MSG_CARD_MENU                       "Imprimir do SD"
+#define MSG_NO_CARD                         "Sem cartao SD"
+#define MSG_DWELL                           "Em suspensao..."
+#define MSG_USERWAIT                        "Espera utilizad."
+#define MSG_RESUMING                        "Retomar impressao"
+#define(length=20) MSG_PRINT_ABORTED        "Impress. abortada"
+#define MSG_NO_MOVE                         "Sem movimento."
+#define MSG_KILLED                          "CANCELADA. "
+#define MSG_STOPPED                         "PARADA. "
+#define MSG_FILAMENTCHANGE                  "Trocar filamento"
+#define MSG_INIT_SDCARD                     "Iniciar cartao SD"
+#define MSG_CNG_SDCARD                      "Trocar cartao SD"
+#define MSG_BABYSTEP_X                      "Micropassos X"
+#define MSG_BABYSTEP_Y                      "Micropassos Y"
+#define MSG_BABYSTEP_Z                      "Ajuste ao vivo Z"
+#define MSG_ADJUSTZ							"Auto ajustar Z?"
+#define MSG_PICK_Z							"Escolha impressao"
+
+#define MSG_SETTINGS                         "Definicoes"
+#define MSG_PREHEAT                         "Pre aquecimento"
+#define(length=17) MSG_UNLOAD_FILAMENT               "Descargar filam."
+#define(length=17) MSG_LOAD_FILAMENT                 "Carga filamento"
+#define(length=17) MSG_LOAD_FILAMENT_1				"Carga filamento 1"
+#define(length=17) MSG_LOAD_FILAMENT_2				"Carga filamento 2"
+#define(length=17) MSG_LOAD_FILAMENT_3				"Carga filamento 3"
+#define(length=17) MSG_LOAD_FILAMENT_4				"Carga filamento 4"
+#define(length=17) MSG_UNLOAD_FILAMENT_1				"Descarga filam. 1"
+#define(length=17) MSG_UNLOAD_FILAMENT_2				"Descarga filam. 2"
+#define(length=17) MSG_UNLOAD_FILAMENT_3				"Descarga filam. 3"
+#define(length=17) MSG_UNLOAD_FILAMENT_4				"Descarga filam. 4"
+#define MSG_UNLOAD_ALL						"Descarregar todos"
+#define MSG_LOAD_ALL						"Carregar todos"
+
+
+#define MSG_RECTRACT                        "Retração"
```

Comment:
Is it allowed to write the 'ç' as 'c' and the 'ã' as 'a'? Or does the meaning change doing that?